pkg/crypto/webkms/remotecrypto.go

```diff
@@ -827,3 +828,54 @@ func closeResponseBody(respBody io.Closer, logger spi.Logger, action string) {
 		logger.Errorf("Failed to close response body for '%s' REST call: %s", action, err.Error())
 	}
 }
+
+// Issuer
+
+// CLGetCredentialDefinition - To be implemented.
+func (r *RemoteCrypto) CLGetCredentialDefinition(kh interface{}) (*cl.CredentialDefinition, error) {
```

Comment:
The intent of the Crypto interface was more at the level of Crypto primitives (sign, verify, encrypt, etc). Similarly the Web KMS operates at the primitives level. I would not expect a system like WebKMS to be implementing functionality at the level of credential definitions, credentials, etc.